apps/nextjs-website/src/components/organisms/WebinarQuestionsTemplate/WebinarQuestionsTemplate.tsx

```diff
@@ -32,20 +34,48 @@ const WebinarQuestionsTemplate = ({
     webinar && setWebinar(webinar);
   }, [webinar]);
 
+  useEffect(() => {
+    setQuestions(data ?? []);
+  }, [data]);
+
   if (error || (!loading && !user)) return <PageNotFound />;
   else if (!data || loading || !user) return <Spinner />;
   else {
     const userName = `${user.attributes['given_name']} ${user.attributes['family_name']}`;
-    const sortedVisibleQuestions = [...data]
+    const sortedVisibleQuestions = [...questions]
       .sort((a, b) => b.id.createdAt.getTime() - a.id.createdAt.getTime())
       .filter(({ hiddenBy }) => !hiddenBy || userName === hiddenBy);
+
     const highlightedQuestions = sortedVisibleQuestions.filter(
       (question) => !!question.highlightedBy
     );
+
     const notHighlightedQuestions = sortedVisibleQuestions.filter(
       (question) => !question.highlightedBy
     );
 
+    const updateLocalQuestions = (
```

Comment:
I would name this differently, because it is not clear form the parametrers that questions are selected by date and then updated, it seems like you are updating all 3 params. 

I would use something like 
updateLocalQuestionsByDate(date, updateHide, updateHighlight